packages/expo-notifications/README.md

```diff
@@ -432,16 +431,16 @@ import * as Notifications from 'expo-notifications';
 import { Linking } from 'react-native';
 
 export default function App() {
-  const initialNotificationResponse = Notifications.useInitialNotificationResponse();
+  const lastNotificationResponse = Notifications.useLastNotificationResponse();
   React.useEffect(() => {
     if (
-      initialNotificationResponse &&
-      initialNotificationResponse.notification.data.url &&
-      initialNotificationResponse.actionIdentifier === Notifications.DEFAULT_ACTION_IDENTIFIER
+      lastNotificationResponse &&
+      lastNotificationResponse.notification.data.url &&
```

Comment:
@sjchmiela  I was wondering if this is the correct way to access `notification.data`, since `lastNotificationResponse` contain the response from `addNotificationResponseReceivedListener`. Shouldn't it be `notification.request.content.data` ?